# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/i486.patch
# Copyright (C) 2020 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./third_party/rust/target-lexicon/src/targets.rs.orig	2022-07-18 20:44:31.000000000 +0200
+++ ./third_party/rust/target-lexicon/src/targets.rs	2022-07-26 16:11:48.495907635 +0200
@@ -330,6 +330,7 @@
 #[allow(missing_docs)]
 pub enum X86_32Architecture {
     I386,
+    I486,
     I586,
     I686,
 }
@@ -709,6 +710,7 @@
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let s = match *self {
             X86_32Architecture::I386 => "i386",
+            X86_32Architecture::I486 => "i486",
             X86_32Architecture::I586 => "i586",
             X86_32Architecture::I686 => "i686",
         };
@@ -889,6 +891,7 @@
 
         Ok(match s {
             "i386" => I386,
+            "i486" => I486,
             "i586" => I586,
             "i686" => I686,
             _ => return Err(()),
@@ -1390,6 +1393,8 @@
             "hexagon-unknown-linux-musl",
             "i386-apple-ios",
             "i586-pc-windows-msvc",
+            "i486-unknown-linux-gnu",
+            "i486-unknown-linux-musl",
             "i586-unknown-linux-gnu",
             "i586-unknown-linux-musl",
             "i686-apple-darwin",
--- firefox-103.0/third_party/rust/target-lexicon/.cargo-checksum.json.vanilla	2022-07-26 16:12:02.403907080 +0200
+++ firefox-103.0/third_party/rust/target-lexicon/.cargo-checksum.json	2022-07-26 16:12:34.239905811 +0200
@@ -1 +1 @@
-{"files":{"Cargo.lock":"b9264c19ff518102b65fbe3727bfff730e671546f83c6d1375fb0c1f907710f6","Cargo.toml":"0cfc8c6ade28f12605500e0ec23192094a3e0a9eefcbce850f9a1b5a7652c646","LICENSE":"268872b9816f90fd8e85db5a28d33f8150ebb8dd016653fb39ef1f94f2686bc5","README.md":"c3467056d91be3f59562158ee9604c729b5b5f473efbefb036032803eb76809e","build.rs":"4716b4f955c7a4cb39cb3b7521c1745d5110c1cbd1e054bca906e37f5e974675","examples/host.rs":"503bafddfb372123fe4dc0e7b8037808beb5bfe6df60c00d3315922bd3792c6c","examples/misc.rs":"49a579845450b7b020ed5c97dca142fc548725893cbc82f6f750ee0caab2beca","scripts/rust-targets.sh":"89564342916321c5bc35e772d374a7f0af22cc9ae6dcc0027eca48d2269f18cb","src/data_model.rs":"38c0c184ee65b882872298127d9c6878292bd12c139792e82328a5facc2428ae","src/host.rs":"fb543df4f362e9119a58523563e453110f4e3a426f0995911d0ca386657cf1d9","src/lib.rs":"8e507b48bd616e1fae0e4664770f66a6475b3c6b5377f88edae1e129d133c4eb","src/parse_error.rs":"b3735eabc0fd0a9dfdd6375662f20ec96a79852a00a05a98fb2e421545285e53","src/targets.rs":"b8eea045a1bc8d19e8e5b0cdd969727c8403982062772fab79dc982a7d237470","src/triple.rs":"f609b02ff35df25adae00b0965484043b37bb0e0c1cc5b6aa9729b69f6811879"},"package":"c02424087780c9b71cc96799eaeddff35af2bc513278cda5c99fc1f5d026d3c1"}
\ No newline at end of file
+{"files":{"Cargo.lock":"b9264c19ff518102b65fbe3727bfff730e671546f83c6d1375fb0c1f907710f6","Cargo.toml":"0cfc8c6ade28f12605500e0ec23192094a3e0a9eefcbce850f9a1b5a7652c646","LICENSE":"268872b9816f90fd8e85db5a28d33f8150ebb8dd016653fb39ef1f94f2686bc5","README.md":"c3467056d91be3f59562158ee9604c729b5b5f473efbefb036032803eb76809e","build.rs":"4716b4f955c7a4cb39cb3b7521c1745d5110c1cbd1e054bca906e37f5e974675","examples/host.rs":"503bafddfb372123fe4dc0e7b8037808beb5bfe6df60c00d3315922bd3792c6c","examples/misc.rs":"49a579845450b7b020ed5c97dca142fc548725893cbc82f6f750ee0caab2beca","scripts/rust-targets.sh":"89564342916321c5bc35e772d374a7f0af22cc9ae6dcc0027eca48d2269f18cb","src/data_model.rs":"38c0c184ee65b882872298127d9c6878292bd12c139792e82328a5facc2428ae","src/host.rs":"fb543df4f362e9119a58523563e453110f4e3a426f0995911d0ca386657cf1d9","src/lib.rs":"8e507b48bd616e1fae0e4664770f66a6475b3c6b5377f88edae1e129d133c4eb","src/parse_error.rs":"b3735eabc0fd0a9dfdd6375662f20ec96a79852a00a05a98fb2e421545285e53","src/triple.rs":"f609b02ff35df25adae00b0965484043b37bb0e0c1cc5b6aa9729b69f6811879"},"package":"c02424087780c9b71cc96799eaeddff35af2bc513278cda5c99fc1f5d026d3c1"}
